prompting/utils/exceptions.py

```diff
@@ -4,3 +4,11 @@ class MaxRetryError(Exception):
     def __init__(self, message="Maximum number of retries exceeded"):
         self.message = message
         super().__init__(self.message)
+
+
+class TaskCreationError(Exception):
+    """Exception raised when the task creation fails."""
```

Comment:
Abstraction seems a little bit superfluous - it doesn't hurt anyone to keep it but all it does is set a default message which is only ever used in the tests